bentoml/pytorch.py

```diff
@@ -228,34 +226,34 @@ def _setup(self) -> None:  # type: ignore[override]
                 device_id=self._device_id,
             )
         raw_predict_fn = getattr(self._model, self._predict_fn_name)
-        self._predict_fn: t.Callable[..., _RV] = functools.partial(
+        self._predict_fn: t.Callable[..., torch.Tensor] = functools.partial(
             raw_predict_fn, **self._partial_kwargs
         )
 
-    # pylint: disable=arguments-differ
     @torch.no_grad()
-    def _run_batch(  # type: ignore[override]
+    def _run_batch(
         self,
-        *args: t.Union[np.ndarray, torch.Tensor],
+        *args: t.Union["np.ndarray[t.Any, np.dtype[t.Any]]", torch.Tensor],
         **kwargs: str,
```

Comment:
@larme @aarnphm what's the reason for `**kwargs: str` here?

Reply:
it should be impossible to have an str type here, data container does not know how to aggregate a batch for it